setup.py

```diff
@@ -121,13 +121,13 @@ def read(*names, **kwargs):
         'open source', 'scan', 'license', 'package', 'dependency',
         'copyright', 'filetype', 'author', 'extract', 'licensing',
     ],
-    python_requires='>=2.7,<3',    
+    python_requires='>=2.7,<4',    
     install_requires=[
         # cluecode
         # Some nltk version ranges are buggy
         'nltk >= 3.2, < 4.0',
         'publicsuffix2',
-        'py2-ipaddress >= 2.0, <3.5',
+        'ipaddress',
```

Comment:
If you want this merged in develop, I will need to have a pre-built wheel created and contributed. Also there is may be a better way since ipaddress is builtin 3.6.x: you can keep the dep on `py2-ipaddress` and make it conditional on being on Python2 only. That would be easier to merge.